README.md

```diff
@@ -1,24 +1,146 @@
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* System dependencies
+<!-- PROJECT DESCRIPTION -->
 
-* Configuration
+# 📖 Blog <a name="about-project"></a>
 
-* Database creation
+This project is a Ruby on Rails app for writing blogs where users can login to write and read blogs
 
-* Database initialization
+## 🛠 Built With <a name="built-with"></a>
 
-* How to run the test suite
+### Tech Stack <a name="tech-stack"></a>
+- The app built using [Ruby on Rails](https://www.ruby-lang.org/en/).
 
-* Services (job queues, cache servers, search engines, etc.)
+<!-- Features -->
 
-* Deployment instructions
+### Key Features <a name="key-features"></a>
 
-* ...
+The app help you to keep record of written blogs like
+
+- **Store and Display Blogs**
+- **Store and writers**
+- **Store likes and comments**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps...
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Ruby should be installed on your local machine
+- Rails should be installed on your local machine
+- You need a Terminal.
+- You need a code editor eg: `VsCode`
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone https://github.com/LDouglasOT/BlogApp.git
+```
+
+### Install
+
+Install this project with:
+
+Example command:
+
+```sh
+  cd ruby-blog
+  bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+Example command:
+
+```sh
+  rails s
+```
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤**Daniel Kigozi**
+
+- GitHub: [@githubhandle](https://github.com/LDouglasOT)
+- Twitter: [@twitterhandle]()
+- LinkedIn: [LinkedIn]()
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Add more features**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/LDouglasOT/BlogApp/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project kindly give it a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank [Microverse](https://www.microverse.org/) for giving this opportunity to learn.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- Kindly, your `mit` link is broken, so please update it to have a better user experience 👍